.circleci/config.yml

```
@@ -219,6 +223,16 @@ jobs:
           name: Shellcheck Lint
           command: yarn lint:shellcheck
 
+  test-integration-storybook:
+    docker:
+      - image: koalaman/shellcheck-alpine@sha256:169a51b086af0ab181e32801c15deb78944bb433d4f2c0a21cc30d4e60547065
+    steps:
+      - checkout
+      - run: apk add --no-cache bash yarn
```

This is missing this step:
```suggestion
      - attach_workspace:
          at: .
      - run: apk add --no-cache bash yarn
```

As a result `storybook` is not present at the moment.